library/alloc/src/collections/btree/set.rs

```diff
@@ -1232,6 +1232,7 @@ impl<T: Ord, const N: usize> From<[T; N]> for BTreeSet<T> {
     }
 }
 
+/// Gets an owning iterator over the elements of the `BTreeSet` in ascending order.
```

Comment:
I think it's more useful to put this on the method

Reply:
this one has into_iter docs already so i just appended the ascending order bit to that, however map has no docs on into_iter, so i just copied the fulltext